JapaneseFilter/sections/specific.txt

```diff
@@ -10,6 +10,7 @@ javkun.com##.adsvideo
 69re.tv###wrapper > div.container > div[style="width: 100%"] > a[target="_blank"] > img
 hosyusokuhou.jp###side > div.sidewrapper[style^="width: 218px; margin-bottom: 0px; position:"] a[target="_blank"] > img
 hosyusokuhou.jp##.abanner
+macsoft.jp##td[align="center"] > .flame[width="230px"]
```

Comment:
Please use an ext-CSS rule (which I described in previous comment). Site is using .flame for valid content too and only [width] selector is too generic.